ApollosProject/apollos-prototype
# Move ShareContentButtonConnected into the ui-connected package

## DESCRIPTION

<img src="https://user-images.githubusercontent.com/2528817/73002826-3b8ce900-3dca-11ea-93c2-5b57aec573e1.png" width="30%" />

### What does this PR do, or why is it needed?

Move ShareContentButtonConnected into the ui-connected package.

### How do I test this PR?

* Open app
* Click on article/card
* Click share button and verify it works correctly

### What automated tests did you write?
Same just relocated.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
